htdocs/core/lib/images.lib.php

```diff
@@ -77,7 +77,10 @@ function dol_getImageSize($file, $url = false)
 	$fichier = $file;
 	if (!$url)
 	{
-		$fichier = realpath($file); 	// Chemin canonique absolu de l'image
+		$fichier = realpath(dol_osencode($file)); 	// Chemin canonique absolu de l'image
```

Comment:
If you need dol_osencode of $file, you probably need it 3 lines later 
$dir = dirname($file);
don't you ?